### PR TITLE
Add convention tests directory location switch

### PR DESCRIPTION
**Pull request by kokosing**
Add convention tests directory location switch

Issue: https://github.com/prestodb/tempto/issues/106 // I (@maciejgrzybek) edited comment adding this line to allow better tracking

### CODE REVIEW

**Comment by losipiuk** [comment]
looks good

File: tempto-core/src/main/java/com/teradata/tempto/internal/convention/ConventionTestsUtils.java [hunk 1]
```diff
@@ -53,7 +55,12 @@
     public static Optional<Path> getConventionsTestsPath(String child)
     {
         try {
-            Enumeration<URL> productTestUrls = getSystemResources(CONVENTION_TESTS_ROOT_DIR + "/" + child);
+            String conventionTestsDir = System.getProperty(CONVENTION_TESTS_DIR_KEY);
+            if (new File(conventionTestsDir).exists()) {
+                return Optional.of(Paths.get(conventionTestsDir));
```
Comment (losipiuk) [inline tempto-core/src/main/java/com/teradata/tempto/internal/convention/ConventionTestsUtils.java]: copy is needed because we run scripts in copied path.

**Comment by maciejgrzybek** [comment]
Documentation change is needed to describe new config property and to state the precedence of filesystem path over resources.

**Comment by kokosing** [comment]
@losipiuk @maciejgrzybek I addressed comments, can you please take a look again?

File: README.md [hunk 2]
```diff
@@ -811,14 +827,15 @@ By default all tests found in classpath are executed but user may limit that.
             looks like: sql_tests.testcases.sample_table.allRows. Tests which name ends with one of patterns specified
             in --tests parameter will be executed.
         </td>
+        <td>All tests are selected.</td>
     </tr>
     <tr>
         <td>--exclude-groups</td>
         <td>List of test groups which should be excluded from execution.</td>
+        <td>None groups are excluded.</td>
```
Comment (losipiuk) [inline README.md]: No instead of none

**Comment by losipiuk** [comment]
looks good

**Pull request merged**